auditlog/diff.py

```diff
@@ -1,6 +1,6 @@
 from django.conf import settings
 from django.core.exceptions import ObjectDoesNotExist
-from django.db.models import NOT_PROVIDED, DateTimeField, Model
+from django.db.models import NOT_PROVIDED, DateTimeField, JSONField, Model
```

Comment:
You should prefer `from django_jsonfield_backport.models import JSONField` instead, for Django compat'.

Reply:
Also updated this one.